src/cryptonote_core/service_node_rules.h

```diff
@@ -2,29 +2,30 @@
 
 #include "crypto/crypto.h"
 #include "cryptonote_config.h"
-#include "service_node_voting.h"
-
-#include <random>
+#include "cryptonote_core/service_node_voting.h"
 
 namespace service_nodes {
-  constexpr size_t   DEREGISTER_QUORUM_SIZE                    = 10;
+  constexpr size_t   UPTIME_QUORUM_SIZE                     = 10;
```

Comment:
I don't like the name "uptime quorum" for this (see my earliest comment).  I called it `state_change` in my PR, but I don't know if I like that, either.

The quorum here is fundamentally about policing for bad behaviour, so many "police_quorum" or "behaviour_quorum" or something along those lines?

Reply:
The way I come about this is, what are we validating, uptime proofs, what do the workers need to do, generate uptimes, so uptime quorum.

Now we're validating that uptime proofs and checkpoints have been voted, so umbrella term like behaviour or participation sound fine. But uptime quorum is obviously going to go away since it's going to be based off more than one criteria now.